Append default CFLAGS to the ones MacPorts sets; increases performance.
--- configure.in.orig	2014-04-08 13:41:05.000000000 -0700
+++ configure.in	2014-06-13 07:49:59.000000000 -0700
@@ -119,7 +119,7 @@
 dnl line
 dnl
 AC_MSG_CHECKING(whether to use default CFLAGS)
-if test  x"$CFLAGS" = x"" ; then
+if true; then
    force_default_cc_options="yes"
    AC_MSG_RESULT([yes])
 else
@@ -694,7 +694,7 @@
 dnl ==========================================================================
 
 if test "$force_default_cc_options" = "yes" ; then
-   CFLAGS="$our_cflags_defaults"
+   CFLAGS+=" $our_cflags_defaults"
 fi
 
 dnl ==========================================================================
